### DISCUSSION TITLE
Would love some help on running test process

### DISCUSSION THREAD

**Discussion by zdcthomas** [Platonic-Systems/process-compose-flake#50] [Q&A]
Hey! I've been really loving this project and it's been super helpful! I'm trying to figure out, though, how to run the `test` process in CI, I've tried using `nixci` and nix check, but they don't pick up the test. When I run the process compose with `nix run`, I can run the test manually, but it doesn't get called when I run `nix flake check.
I can provide whatever info is helpful! Thanks!

**Comment by srid** [comment]
You can check out one approach in https://github.com/juspay/services-flake - where we just use a sub-flake with override of parent flake.

**Reply by srid** [reply]
https://github.com/juspay/services-flake/blob/main/test/flake.nix

**Reply by zdcthomas** [reply]
Ah interesting, maybe I was running nixci wrong, I'll look into that. Thanks! Is there a way to include the check without nixci?

**Reply by srid** [reply]
I'm pretty sure nixci is not required, and `nix flake check` should work. Feel free to send a detailed and reproducible bug report if you think there's a bug.

**Reply by zdcthomas** [reply]
Ok awesome, let me make a smaller reproducible example and if that causes the same thing I'll post a bug report. Thanks!
